grafana/resource_data_source.go

```diff
@@ -71,6 +71,12 @@ func ResourceDataSource() *schema.Resource {
 				Sensitive: true,
 			},
 
+			"tls_skip_verify": {
```

Comment:
(see other comment below too) I think it might be better to have this be an attribute of `json_data` since that's how it was implemented in the upstream library → https://github.com/nytm/go-grafana-api/pull/13/files#diff-4fadc66382f2a85543cefe051bedef99R39

Reply:
Hey, @mlclmj, I paired a bit with @cwcordell on this and can weigh in on why the attribute was placed where it was:

We started out having the same thought as you and initially looked at putting it in the `json_data` attribute, but while that attribute is optional, if you provide it, you *must* provide 2 of the keys defined within it: `auth_type`, `default_region`.  We didn't see an obvious way to make those those keys optional while ensuring that if one of them is set, the other is.

Because of that, we decided that putting this as a separate attribute made more sense and it more closely aligns with the Grafana UI, where "Skip TLS Verify" is an option at the same level as "Basic Auth", etc.

If you're good with the above rationale and are otherwise okay with this PR, I can make sure we get the docs updated as part of this as well.

Thanks!

Reply:
Colin explained it well.